spec/css-spec.coffee

```diff
@@ -20,6 +20,19 @@ describe 'CSS grammar', ->
       expect(tokens[8]).toEqual value: ';', scopes: ['source.css', 'meta.property-list.css', 'meta.property-value.css', 'punctuation.terminator.rule.css']
       expect(tokens[10]).toEqual value: '}', scopes: ['source.css', 'meta.property-list.css', 'punctuation.section.property-list.end.css']
 
+    # Needs more complex examples
```

Comment:
What more complex examples are needed?  Any chance you can add them onto this PR?